packages/fxa-settings/src/models/hooks.ts

```diff
@@ -118,17 +118,19 @@ export function useExperiments(): NimbusResult | null {
     async function fetchExperiments() {
       if (experimentInfo) {
         const exp = await experimentInfo;
```

Comment:
Agree we shouldn't set experiments if there is nothing to set.
This setup seems odd to me though, as well as the loose typing that's caused us to miss that `features` should be lower case. Threw up a quick branch for consideration: 
https://github.com/mozilla/fxa/compare/nimbus-experiment-fetch-refactor?expand=1

Reply:
I just looked at what comes back from the server and it should in fact be `exp.Features`... 

![image](https://github.com/user-attachments/assets/32a050ac-de4f-4792-be2f-0a313dac68cb)


I agree the any typing isn't great though. I'll take a look at your branch and see I can fix the typing.

Reply:
I looked at your branch further. I think the additional awaits are not desirable. I believe the code was structured without awaits intentionally so as not block loading. I think there should be a better way to get experiments that doesn't involve an additional round trip or more blocking behavior. However, I think this is outside the scope of the Sentry issue I'm trying to address here. I'd like to approach this incrementally, and will file a follow up for reviewing how we get the actual experiment state.